components/x-topic-search/readme.md

```diff
@@ -1,6 +1,6 @@
 # x-topic-search
 
-This module provides a topic search bar.
+This module provides a topic search bar. It displays topic suggestions which are fetched from the api, passed as a property, based on user input(search term). x-follow-button is displayed with the topic suggested.
```

Comment:
Could we link/reference the API it uses?

Reply:
Also might be worth rewording the last sentence to suggest there are multiple topics:

```suggestion
This module provides a topic search bar. It displays topic suggestions which are fetched from the api, passed as a property, based on user input(search term). `x-follow-button`s are used for each of the suggestions to allow users to follow directly
```